source/Playnite.DesktopApp/ViewModels/DesktopAppViewModel.cs

```diff
@@ -593,6 +648,7 @@ public async void ImportEmulatedGames(EmulatedGamesImportViewModel model)
                     Logger.Warn("Skipping metadata download for manually added emulated games, some global task is already in progress.");
                 }
             }
+            await SetSortingNames(model.ImportedGames, true);

```

Comment:
This will cause sorting names to be applied twice when UpdateEmulationLibrary is used via UpdateLibraryData.

Reply:
Could you doublecheck this? I can't find a call to DesktopAppViewModel.ImportEmulatedGames from UpdateLibraryData. You might be thinking of MainViewModelBase.ImportEmulatedGames. Maybe this call to SetSortingNames should be in MainViewModelBase though.